tools/spec-gen-sdk/src/automation/workflowPackage.ts

```diff
@@ -159,7 +159,7 @@ const workflowPkgSaveApiViewArtifact = async (context: WorkflowContext, pkg: Pac
   context.sdkApiViewArtifactFolder = destination;
   console.log(`##vso[task.setVariable variable=HasApiViewArtifact]true`);
   const artifactName = `sdkApiViewArtifact_${language}`; // it's the artifact in pipeline artifacts
-  console.log(`##vso[task.setVariable variable=ArtifactName]${artifactName}`);
```

Comment:
I don't think we need to set this variable here, since the publish is happening on the pipeline. 
I think what needs to be set as a variable is the path to staged artifacts

Reply:
I agree that we could remove this variable and construct the name in the pipeline since now the pipeline knows the language by parsing the SDK repo name. Can you include this change as part of your PR for the API view meta file content change?